php/commands/auth.php

```diff
@@ -89,7 +89,7 @@ public function login( $args, $assoc_args ) {
    */
   public function logout() {
     Terminus::line( "Logging out of Pantheon." );
-    Terminus::launch_self("cli",array('cache-clear'));
+    $this->cache->flush(null);
```

Comment:
I don't know PHP style at all, so maybe this is a bad question.

Why pass `null` here? It looks like that's the default parameter anyways?
`public function flush($cache=null,$exclude=null) {` ...